elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/timegrains/DateTime.java

```diff
@@ -5,7 +5,7 @@
  */
 package com.yahoo.elide.datastores.aggregation.timegrains;
 
-import java.util.Date;
+import java.sql.Date;
```

Comment:
We should use sql.Time or Timestamp because java.sql.date does not store Time Portion of the date.

Reply:
updated.